src/main/java/core/basesyntax/MainApp.java

```diff
@@ -11,6 +11,17 @@
  * - В классе Main создать добавить по одному екземпляру каждого класса.
  * - Отправить все машини на работу и остановить работу всех машин.
  */
-public class MainApp {
 
+public class MainApp {
+    public static void main(String[] args) {
+        Track t = new Track();
+        Bulldozer b = new Bulldozer();
+        Excavator e = new Excavator();
```

Comment:
```suggestion
        Machine t = new Track();
        Machine b = new Bulldozer();
        Machine e = new Excavator();
```
Also, t,b,e - not informative naming

Reply:
done